source/api/image/3.0/change-log.md

```diff
@@ -131,17 +141,33 @@ All examples now use `https` URIs in order to reflect best practices for interop
 
 #### 3.5. Clarify and move discussion of floating point representation
 
-Discussion of floating point representation has been moved from the image requests introduction to the [Canonical URI Syntax][image30-canonical-uri-syntax] section, and has been expanded for clarity. The advice on using at most 10 decimal digits has been removed. See issue [#1468](https://github.com/IIIF/api/issues/1468).
+Discussion of floating point representation has been moved from the image requests introduction to the [Canonical URI Syntax][image30-canonical-uri-syntax] section, and has been expanded for clarity. The advice on using at most 10 decimal digits has been removed. Description of `pct` values clarifies these may be either floating point or integer. See issues [#1468](https://github.com/IIIF/api/issues/1468) and [#1240](https://github.com/IIIF/api/issues/1240).
 
 #### 3.6. Clarify support for CORS
 
 Discussion of CORS has be moved to a new [CORS][image30-cors-response] section and the wording clarified to make it clear that not only the `Access-Control-Allow-Origin` header but also the preflight request pattern (ie. including the `OPTIONS` header) should be supported. See issue [#1274](https://github.com/IIIF/api/issues/1274).
 
-#### 3.7. Implementation with static web resources
+#### 3.7. Clarify that the `@context` value ay be a single URI or an array or URIs
+
+The previous version mentioned only the Image API JSON-LD context. The description has been changed to make it clear that the value of `@context` may be either a single URI, or an array with extension context URIs following the Image API context URI. See issue [#1472](https://github.com/IIIF/api/issues/1472).
```

Comment:
Extensions contexts must be before the Image API context

Reply:
:+1: Good catch! Would have been bad to have that the wrong way around.